src/gui/transferlistfilterswidget.cpp

```diff
@@ -363,18 +363,17 @@ TrackerFiltersList::TrackerFiltersList(QWidget *parent, TransferListWidget *tran
 {
     auto *allTrackers = new QListWidgetItem(this);
     allTrackers->setData(Qt::DisplayRole, tr("All (0)", "this is for the tracker filter"));
-    allTrackers->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"network-server"_qs));
+    allTrackers->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"trackers"_qs));
     auto *noTracker = new QListWidgetItem(this);
     noTracker->setData(Qt::DisplayRole, tr("Trackerless (0)"));
-    noTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"network-server"_qs));
+    noTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"trackerless"_qs));
     auto *errorTracker = new QListWidgetItem(this);
     errorTracker->setData(Qt::DisplayRole, tr("Error (0)"));
-    errorTracker->setData(Qt::DecorationRole, style()->standardIcon(QStyle::SP_MessageBoxCritical));
+    errorTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"tracker-error"_qs));
     auto *warningTracker = new QListWidgetItem(this);
     warningTracker->setData(Qt::DisplayRole, tr("Warning (0)"));
-    warningTracker->setData(Qt::DecorationRole, style()->standardIcon(QStyle::SP_MessageBoxWarning));
-
-    m_trackers[NULL_HOST] = {{}, noTracker};
+    warningTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon(u"tracker-warning"_qs));
+    m_trackers[NULL_HOST] = {};
```

Comment:
Revert this line. It causes a crash.

Reply:
@xavier2k6 How to handle this?

Reply:
Was it an intended change? If not then just change `m_trackers[NULL_HOST] = {};` back to `m_trackers[NULL_HOST] = {{}, noTracker};`

Reply:
It was intended, we `tracker-warning` is required there.

Reply:
> It was intended, we `tracker-warning` icon is required there.

But what does the discussed line have to do with it?

Reply:
@xavier2k6 Only this issue remains.

Reply:
Committed the change in the PR.

Reply:
>@thalieht when these changes are made can you test to see if it still then crashes for you?

I already have, that's how i know this was causing it 👍